core/account/builder_test.go

```diff
@@ -22,11 +22,19 @@ import (
 
 func TestAccountSourceReserve(t *testing.T) {
 	var (
-		_, db    = pgtest.NewDB(t, pgtest.SchemaPath)
-		ctx      = pg.NewContext(context.Background(), db)
-		c        = prottest.NewChain(t)
+		_, db = pgtest.NewDB(t, pgtest.SchemaPath)
+		ctx   = pg.NewContext(context.Background(), db)
+		c     = prottest.NewChain(t)
+	)
+
+	b1, err := c.GetBlock(ctx, 1)
```

Comment:
Would it be reasonable for prottest to provide a function to do this lookup and Fatal the test if it's not found? This snippet here crops up in enough places now it would probably pay for itself pretty easily.